packages/plugin-response-cache/src/ApolloServerPluginResponseCache.ts

```diff
@@ -0,0 +1,331 @@
+import type {
+  ApolloServerPlugin,
+  BaseContext,
+  CacheHint,
+  GraphQLRequestContext,
+  GraphQLRequestListener,
+  GraphQLResponse,
+} from '@apollo/server';
+import { createHash } from '@apollo/utils.createhash';
+import {
+  KeyValueCache,
+  PrefixingKeyValueCache,
+} from '@apollo/utils.keyvaluecache';
+
+type ValueOrPromise<T> = T | Promise<T>;
+interface Options<TContext = Record<string, any>> {
```

Comment:
don't we want to export this? (in which case maybe we give it a longer name)

Reply:
Yep good call! [f39fef9](https://github.com/apollographql/apollo-server/pull/6610/commits/f39fef999ddba78e01d9641a2fd1a8e6a874a38e)